### DISCUSSION TITLE
[@types/formidable] missing option { maxFiles: number }

### DISCUSSION THREAD

**Discussion by robahtou** [DefinitelyTyped/DefinitelyTyped#58015] [Issues with a @types package]
Formidable allows for the option `maxFiles` as seen here and in their options:
https://github.com/node-formidable/formidable/blob/master/src/Formidable.js#L23
https://github.com/node-formidable/formidable#options

The option type is missing. Suggest the following:
```
/**
   * limit the amount of uploaded files
   *
   * @default Infinity
 */
        maxFiles?: number | undefined;
```

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready #58051
